packages/components/src/disclosure/src/Disclosure.css

```diff
@@ -15,9 +15,9 @@
 
 /* CONTENT */
 .o-ui-disclosure-content-inner {
-    padding-top: var(--o-ui-sp-5);
-    padding-bottom: var(--o-ui-sp-5);
-    color: var(--o-ui-text-alias-primary);
+    padding-top: var(--hop-space-inset-md);
```

Comment:
are we not doing the style like we see in figma? 
![image](https://github.com/gsoft-inc/wl-orbiter/assets/110498164/b17ea805-4fb6-4065-9418-6c4bec24e4bd)

Reply:
that would be a "bordered" accordion component to orbit: 
https://orbit.sharegate.design/?path=/docs/accordion--default-story#bordered

Reply:
Did you already talk to design about this? That seems to be a different component that yes would probably use the same styling.

Reply:
I suppose it would end up being a big change for orbit. It's just a bit confusing.